akka-docs/src/main/paradox/testing.md

```diff
@@ -815,7 +815,7 @@ to the @ref:[reference configuration](general/configuration.md#config-akka-testk
 
 ## Example
 
-Ray Roestenburg's example code from [his blog](http://roestenburg.agilesquad.com/2011/02/unit-testing-akka-actors-with-testkit_12.html) adapted to work with Akka 2.x.
```

Comment:
I've asked @RayRoestenburg about what to do about the link, though I think the drop of the link is perhaps fine... 
Though we could link to the archive post if we wanted to: https://web.archive.org/web/20180114133958/http://roestenburg.agilesquad.com/2011/02/unit-testing-akka-actors-with-testkit_12.html

Reply:
Waiting for the reply then 🙂

Reply:
I will rephrase it stating that this is an archive link and add the link to it.

Reply:
Updated